scripts/plots/global_ens/exevs_global_ens_atmos_gefs_grid2obs_separate_plots.sh

```diff
@@ -264,9 +265,9 @@ for ihr in 00z 12z ; do
         else
             evs_graphic_domain=$domain
         fi
-        for lead in 120 240 360; do
-            mv performance_diagram_regional_${domain}_init_${ihr}_cape_f${lead}__ge250ge500ge1000ge2000.png  evs.global_ens.ctc.cape_l0.last${past_days}days.perfdiag_${valid_time}_f${lead}.g212_buk_${evs_graphic_domain}.png
-        done #lead
+        if [ -f "lead_average_regional_${domain}_init_${ihr}_cape_${stats}_${thresh}.png" ]; then
+            mv lead_average_regional_${domain}_init_${ihr}_cape_${stats}_${thresh}.png evs.global_ens.${stats}_${thresh}.cape_l0.last${past_days}days.fhrmean_init${ihr}_f384.g212_buk_${evs_graphic_domain}.png
+        fi
```

Comment:
@MarcelCaron-NOAA This didn't get updated correctly. The whole renaming of the performance diagram got replaced with something about lead averages.

Reply:
```
for lead in 120 240 360; do
            mv performance_diagram_regional_${domain}_init_${ihr}_cape_f${lead}__ge250ge500ge1000ge2000.png  evs.global_ens.ctc.cape_l0.last${past_days}days.perfdiag_init${ihr}_f${lead}.g212_buk_${evs_graphic_domain}.png
        done #lead
```

Reply:
> @MarcelCaron-NOAA Another change
> 
> In scripts/plots/global_ens/exevs_global_ens_atmos_gefs_grid2obs_separate_plots.sh, please change line 270 `mv performance_diagram_regional_${domain}_init_${ihr}_cape_f${lead}__ge250ge500ge1000ge2000.png evs.global_ens.ctc.cape_l0.last${past_days}days.perfdiag_${valid_time}_f${lead}.g212_buk_${evs_graphic_domain}.png` to `mv lead_average_regional_${domain}_init_${ihr}_cape_${stats}_${thresh}.png evs.global_ens.${stats}_${thresh}.cape_l0.last${past_days}days.fhrmean_init${ihr}_f 384.g212_buk_${evs_graphic_domain}.png`

This isn't the change needed on that line?

Reply:
oops I see your comment here.  Let me correct and update.  Thanks

Reply:
Oh I see!

Reply:
It is I who copied the wrong info.

Reply:
[Updated!  ](https://github.com/NOAA-EMC/EVS/pull/348/commits/1091be67ba6eeacdd1013c9407f20d9b5a347584)  Let me know if that looks wrong.

Reply:
Looks good now! Sorry about that.

Reply:
No worries!